interop/xds/client/client.go

```diff
@@ -163,6 +213,49 @@ func (s *statsService) GetClientStats(ctx context.Context, in *testpb.LoadBalanc
 	}
 }
 
+func (s *statsService) GetClientAccumulatedStats(ctx context.Context, in *testpb.LoadBalancerAccumulatedStatsRequest) (*testpb.LoadBalancerAccumulatedStatsResponse, error) {
+	return accStats.buildResp(), nil
+}
+
+type configureService struct {
+	testpb.UnimplementedXdsUpdateClientConfigureServiceServer
+}
+
+func (s *configureService) Configure(ctx context.Context, in *testpb.ClientConfigureRequest) (*testpb.ClientConfigureResponse, error) {
+	types := make(map[string]bool)
+	for _, typ := range in.GetTypes() {
+		types[typ.String()] = true
+	}
+	rpcsToMD := make(map[string][]string)
+	for _, md := range in.GetMetadata() {
+		enumType := md.GetType().String()
+		if !types[enumType] {
+			continue
+		}
+		var rpcType string
+		switch enumType {
+		case testpb.ClientConfigureRequest_UNARY_CALL.String():
+			rpcType = unaryCall
+		case testpb.ClientConfigureRequest_EMPTY_CALL.String():
+			rpcType = emptyCall
+		default:
+			return nil, fmt.Errorf("unsupported RPC type: %v", enumType)
+		}
+		rpcsToMD[rpcType] = append(rpcsToMD[rpcType], md.GetKey(), md.GetValue())
+	}
+	cfgs := make([]*rpcConfig, 0)
```

Comment:
This is the same as `make([]*rpcConfig`.
If you want to pre-allocate space, do `make([]*rpcConfig, 0, len(rpcsToMD)`.

Reply:
Made this change. I get an error with just `make([]*rpcConfig)` (assuming it expects me to just use `var cfgs []*rpcConfig`?) so I did the latter.